distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestPlatform.java

```diff
@@ -97,9 +97,13 @@ public void addConfigurationFile() throws IOException {
         public void addConfigurationFileAndWait(ConfigurationAdmin configAdmin) throws Exception {
             addConfigurationFile();
 
-            expect("Configuration properties for PID " + pid + " to be set").within(20, SECONDS)
-                    .until(new GetConfigurationProperties(configAdmin, "id", pid),
-                            equalToConfigurationProperties(getFileProperties()));
+            try {
+                expect("Configuration properties for PID " + pid + " to be set").within(20, SECONDS)
+                        .until(new GetConfigurationProperties(configAdmin, "id", pid),
+                                equalToConfigurationProperties(getFileProperties()));
+            } catch (ConfigurationFileException e) {
```

Comment:
What was the finding here? If the exception is thrown we probably want to fail the test and letting the exception go was the easiest way to do that.

A more general question is, should we even run findbugs on unit and integration tests?